pkg/network/node/networkpolicy.go

```diff
@@ -281,6 +289,38 @@ func (np *networkPolicyPlugin) syncNamespaceFlows(npns *npNamespace) {
 			}
 		}
 
+		ipBlockIndex := 1
+		// ipBlocks with Except rules are validated in Table 81
+		otx.AddFlow("table=80, priority=1, reg1=%d, actions=load:%d->NXM_NX_REG3[],goto_table:81", npns.vnid, ipBlockIndex)
+
+		for _, npp := range npns.policies {
+			for _, ipBlock := range npp.ipBlocks {
+				priority := 100
+				// Default low-priority rule for the current ipBlock set. This rule is hit if traffic does not match
+				// the current ipBlock. So, we reiterate to see if it matches some other ipBlock set.
+				otx.AddFlow("table=81, priority=%d, reg1=%d, reg3=%d, actions=load:%d->NXM_NX_REG3[],resubmit:81", priority, npns.vnid, ipBlockIndex, ipBlockIndex+1)
+				// Each flow in a given ipBlock has a unique priority
+				priority++
```

Comment:
We start with lower ipBlockIndex, and we evaluate ipBlocks from lower to higher ipBlockIndex, therefore I think the lowest ipBlockIndex should get the highest priority.

Also reading this is counter intuitive because the flows are defined in reverse order than they are processed.

Can we start with a very high priority and lower as we go through the loop?